### PR TITLE
Feature/#55 create workflow guidelines

### PR DESCRIPTION
**Pull request by LarssonFilip**
Added Workflow guidelines

### CODE REVIEW

**Review by ViktorM02 — COMMENTED** [review]
This should be following markdown format, you can use a tool to view the resulting markdown format in a markdown viewer. Heres how the format turns out as it is: 
image.

I hope that you can se my screenshot :))

File: documentaion/bms-charger communication specification.md
Comment (ljungqvistaxel) [inline documentaion/bms-charger communication specification.md:1]: Good change, but maybe write about changes like this in upcoming pull requests 👍🏻

**Review by ljungqvistaxel — APPROVED** [review]
Nice

**Review by ViktorM02 — COMMENTED** [review]
You should not move the communication specifications, as it's being worked on by another developer. This would lead to multiple files I think...?

**Review by ViktorM02 — APPROVED** [review]

**Pull request closed without merging**